packages/mobx-react-lite/__tests__/strictAndConcurrentModeUsingTimers.test.tsx

```diff
@@ -2,21 +2,22 @@ import "./utils/killFinalizationRegistry"
 import { act, cleanup, render } from "@testing-library/react"
 import * as mobx from "mobx"
 import * as React from "react"
-
 import { useObserver } from "../src/useObserver"
 import {
-    forceCleanupTimerToRunNowForTests,
-    resetCleanupScheduleForTests
-} from "../src/utils/reactionCleanupTracking"
-import {
-    CLEANUP_LEAKED_REACTIONS_AFTER_MILLIS,
-    CLEANUP_TIMER_LOOP_MILLIS
-} from "../src/utils/reactionCleanupTrackingCommon"
+    REGISTRY_FINALIZE_AFTER,
+    REGISTRY_SWEEP_INTERVAL
+} from "../src/utils/UniversalFinalizationRegistry"
+import { observerFinalizationRegistry } from "../src/utils/observerFinalizationRegistry"
+import { TimerBasedFinalizationRegistry } from "../src/utils/UniversalFinalizationRegistry"
+
+expect(observerFinalizationRegistry).toBeInstanceOf(TimerBasedFinalizationRegistry)
```

Comment:
Same as above

Reply:
> What does it do if it fails? 

Throws?

> it's not inside any test

Does it matter?

It's mainly intented to assert environment (bug in the test), rather than testing that `observerFinalizationRegistry` resolves to correct type (bug in the actual tested code).
Arguably I could split these into two, assertion to check `globalThis` for (not) being `FinalizationRegistry` and then an actual `test` for `observerFinalizationRegistry`. However I don't see much value in doing so.